nikola/plugins/misc/taxonomies_classifier.py

```diff
@@ -44,14 +44,14 @@ class TaxonomiesClassifier(SignalHandler):
     name = "render_taxonomies"
 
     def _do_classification(self, site):
-        # Get list of enabled taxonomy plugins
-        taxonomies = [p.plugin_object for p in site.plugin_manager.getPluginsOfCategory('Taxonomy')]
-        taxonomies = [taxonomy for taxonomy in taxonomies if taxonomy.is_enabled()]
-        # Prepare classification and check for collisions
+        # Needed to avoid strange errors during tests
+        if site is not self.site:
```

Comment:
I think this is because of `blinker`: it doesn't have Nikola-instance-specific storage. So if you initialize a site twice in the same process, you end up having two copies of the plugin which are informed by the same signal. That's hard to fix, if there isn't a possibility to flush all listeners for blinker. And that's something test specific.